src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -575,14 +587,16 @@ public ChannelData<T> flush() {
    * @param formattedInputColumnNames list of input column names after formatting
    * @param insertRowIndex Index of the rows given in insertRows API. Not the same as
    *     bufferedRowIndex
+   * @param error
```

Comment:
pl fill this in (and also why an error object is needed in addRow)

Reply:
Added description.